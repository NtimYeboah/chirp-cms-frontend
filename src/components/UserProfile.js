/**
 * Created by ntimobedyeboah on 4/16/17.
 */
import React, { Component } from 'react';

class UserProfile extends Component {
    render() {
        const image96x96 = 'http://placehold.it/96x96';
        const time = '1 Jan 2016';

        return (
            <div className="media">
                <div className="media-left">
                    <figure className="image is-48x48">
                        <img alt="" src={image96x96} />
                    </figure>
                </div>
                <div className="media-content">
                    <p className="subtitle is-6">John Smith <br />
                        <small>{time}</small>
                    </p>

                </div>
            </div>
        );
    }
}

export default UserProfile;