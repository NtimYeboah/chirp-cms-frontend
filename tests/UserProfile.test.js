/**
 * Created by ntimobedyeboah on 4/19/17.
 */
import React from 'react';
import renderer from 'react-test-renderer';

import UserProfile from './../src/components/UserProfile';

describe('UserProfile snapshot', () => {
    test('it should render component and take snapshot', () => {
        const component = renderer.create( <UserProfile />);

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});