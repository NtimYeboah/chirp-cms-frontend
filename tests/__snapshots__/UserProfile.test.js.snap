exports[`UserProfile snapshot it should render component and take snapshot 1`] = `
<div
  className="media">
  <div
    className="media-left">
    <figure
      className="image is-48x48">
      <img
        alt=""
        src="http://placehold.it/96x96" />
    </figure>
  </div>
  <div
    className="media-content">
    <p
      className="subtitle is-6">
      John Smith 
      <br />
      <small>
        1 Jan 2016
      </small>
    </p>
  </div>
</div>
`;
